styles/youtube/catppuccin.user.css

```diff
@@ -449,6 +449,26 @@
       --ytd-author-comment-badge-icon-color: @text !important;
     }
 
+    /* Skeleton */
+    #guide-skeleton,
+    #home-container-skeleton,
+    #home-chips,
+    #search-container {
+      background-color: @base;
+    }
+
+    .masthead-skeleton-icon,
+    #menu-icon {
+      z-index: 1;
+    }
```

Comment:
Is this needed?

Reply:
Yes, for some reason these two menu items disappear when the ```background-color``` of ```#search-container``` gets changed.
I already tried moving the css blocks and add ```!important``` but I only got it to work with this workaround.